.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "mkosi"]
 	path = mkosi
-	url = https://github.com/systemd/mkosi.git
+	url = https://github.com/sapcc/mkosi.git
 	branch = main
```

Comment:
If you want to include your patch I think you need to use branch v14, there I see a commit from you but on main I don't

Reply:
Fixed it